### ISSUE TITLE
disable tag creation for specific groups

### ISSUE THREAD

**Issue by petrk94** [sismics/docs#323]
Hello,

I would like to suggest the feature to disable the "add new tag" function to specific groups or users.

Thanks

**Comment by jendib** [comment]
What is the use case?

**Comment by petrk94** [comment]
@jendib the use case is, that the admin can disable that function, so that not everyone can mess up the tagging function and make chaos in the structure.

**Comment by jendib** [comment]
But they cannot, the tags are per user (unless you add some permissions, but be default it's only for the current user).

**Comment by petrk94** [comment]
Hmm, for now I have the admin group, there are the tags, the another user is even in the admin group and can access this tags. The tags I created as admin, I want to share with the another group (team), this works, but I want to avoid that they can create own tags, which will cause a mess of organizing.

**Comment by jendib** [comment]
Administrators can see everything indeed. But you shouldn't use admin users for normal usage, and in that case all users have separated tags.

**Comment by petrk94** [comment]
But its possible that a normal user, not connected to a group can make own tags, and this I want to avoid.
So if a user alone is making own tags, its not messing up with the tags which are allowed for the group where the user belongs to?
I just wonder, that it will cause a tag messing up on the admin account, due I dont know whether I can remove the tag a single user created or not.

![757](https://user-images.githubusercontent.com/19719052/58811462-25a29c00-8620-11e9-91bf-8d759a699528.PNG)

**Comment by jendib** [comment]
Yes users can create tags, but they will be the only one to see them (+ admins).
Actually you should never use admin users except for your server configuration, that's the idea.

**Comment by Toucan-Sam** [comment]
I would expect that being able to disable tag creation for users would be to enforce consistency.
For example: If I create a "Vehicles" tag, then my significant other can't go and start putting documents under a new "Cars" tag.

I can see how restricting it would be useful, but it also makes sense to allow individual users to organize things however they want. I've only just started using Teedy, but I guess that my partner could share a document with me and she could have it tagged with "Cars" but then on my account I tag it with "Vehicles" and we each see our own methods of organization.

I agree with petrk94 though. It would be nice to be able to enforce consistency. I can't tell my partner to go look in the "Vehicles" tag if she has the document in a different place. :)

**Comment by jendib** [comment]
Ok I see how it could be useful in this case.

**Comment by arladmin** [comment]
Has there been any further development on this feature?

I'm currently exploring the possibility for Teedy in a business environment, and this feature (creation permission for tags, custom metadata) is a bare essential functionality.

Tags are the equivalent of folders in Teedy and because folder structure (which can be immensely vast) in a business environment is to be kept uniform and centralised, only the admin should have permissions regarding all such administrative features:
- creation, modification, deletion of tags/custom metadata
- access to existing custom metadata
- visibility of users and groups in the system
